tests/src/whisk/core/controller/test/migration/SequenceActionApiMigrationTests.scala

```diff
@@ -0,0 +1,183 @@
+/*
+ * Copyright 2015-2016 IBM Corporation
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package whisk.core.controller.test.migration
+
+import scala.Vector
+
+import org.junit.runner.RunWith
+import org.scalatest.junit.JUnitRunner
+
+import common.TestHelpers
+import common.WskTestHelpers
+import spray.http.StatusCodes.OK
+import spray.httpx.SprayJsonSupport.sprayJsonMarshaller
+import spray.httpx.SprayJsonSupport.sprayJsonUnmarshaller
+import spray.json.DefaultJsonProtocol.RootJsObjectFormat
+import spray.json.DefaultJsonProtocol.listFormat
+import spray.json.DefaultJsonProtocol.StringJsonFormat
+import spray.json.DefaultJsonProtocol.vectorFormat
+import spray.json.JsObject
+import spray.json.pimpAny
+import spray.json.pimpString
+
+import whisk.core.controller.WhiskActionsApi
+import whisk.core.controller.test.ControllerTestCommon
+import whisk.core.entity.AuthKey
+import whisk.core.entity.EntityName
+import whisk.core.entity.EntityPath
+import whisk.core.entity.Exec
+import whisk.core.entity.Parameters
+import whisk.core.entity.Subject
+import whisk.core.entity.WhiskAction
+import whisk.core.entity.WhiskActionPut
+import whisk.core.entity.WhiskAuth
+
+/**
+ * Tests migration of a new implementation of sequences: old style sequences can be updated and retrieved - standalone tests
+ */
+@RunWith(classOf[JUnitRunner])
+class SequenceActionApiMigrationTests extends ControllerTestCommon
+    with WhiskActionsApi
+    with TestHelpers
+    with WskTestHelpers {
+
+    behavior of "Sequence Action API Migration"
+
+    val creds = WhiskAuth(Subject(), AuthKey()).toIdentity
+    val namespace = EntityPath(creds.subject())
+    val collectionPath = s"/${EntityPath.DEFAULT}/${collection.path}"
+    def aname = MakeName.next("seq_migration_tests")
+
+    private def seqParameters(seq: Vector[String]) = Parameters("_actions", seq.toJson)
+
+    it should "list old-style sequence action with explicit namespace" in {
+        implicit val tid = transid()
+        val sequence = Vector("/_/a", "/_/x/b", "/n/a", "/n/x/c").map(stringToFullyQualifiedName(_))
```

Comment:
If I understand the current pipe.js implementation, there is no way you could run it with "just" entity names. pipe.js makes calls directly to whisk, so it needs full names. not sure whether there is any special magic beyond that. Adding @perryibm to confirm.